test/emulation.spec.js

```diff
@@ -218,4 +218,10 @@ module.exports.describe = function({testRunner, expect, playwright, FFOX, CHROMI
     });
   });
 
+  describe('focus', function() {
+    it.skip(!headless)('should think that it is focused by default', async({page}) => {
```

Comment:
Should this be `headless && it(...)`?

Reply:
I don’t think so. Headful is our debugging mode. It ideally would fake focus as well.